[2024-05-31T15:21:56.573-0500] {processor.py:161} INFO - Started process (PID=13437) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:56.574-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:21:56.575-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:21:56.575-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:56.586-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:56.673-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:21:56.673-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:21:56.689-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:21:56.689-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:21:56.716-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.148 seconds
[2024-05-31T15:21:58.040-0500] {processor.py:161} INFO - Started process (PID=13439) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:58.041-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:21:58.044-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:21:58.043-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:58.049-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:21:58.047-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 14
    def ():
        ^
SyntaxError: invalid syntax
[2024-05-31T15:21:58.050-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:21:58.070-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.041 seconds
[2024-05-31T15:22:01.973-0500] {processor.py:161} INFO - Started process (PID=13444) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:01.974-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:01.977-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:01.976-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:01.986-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:01.985-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:01.986-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:02.006-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.045 seconds
[2024-05-31T15:22:04.247-0500] {processor.py:161} INFO - Started process (PID=13446) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:04.248-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:04.250-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:04.250-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:04.257-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:04.256-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:04.258-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:04.277-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.038 seconds
[2024-05-31T15:22:07.015-0500] {processor.py:161} INFO - Started process (PID=13450) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:07.016-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:07.020-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:07.018-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:07.029-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:07.028-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:07.030-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:07.051-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.050 seconds
[2024-05-31T15:22:12.419-0500] {processor.py:161} INFO - Started process (PID=13453) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:12.420-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:12.422-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:12.422-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:12.429-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:12.428-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:12.429-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:12.449-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.036 seconds
[2024-05-31T15:22:16.005-0500] {processor.py:161} INFO - Started process (PID=13457) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:16.006-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:16.010-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:16.009-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:16.024-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:16.023-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:16.024-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:16.048-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.056 seconds
[2024-05-31T15:22:19.456-0500] {processor.py:161} INFO - Started process (PID=13461) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:19.458-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:19.461-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:19.461-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:19.471-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:19.470-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:19.472-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:19.491-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.053 seconds
[2024-05-31T15:22:20.618-0500] {processor.py:161} INFO - Started process (PID=13463) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:20.620-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:20.622-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:20.621-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:20.631-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:20.630-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:20.631-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:20.652-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.045 seconds
[2024-05-31T15:22:21.796-0500] {processor.py:161} INFO - Started process (PID=13465) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:21.798-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:21.801-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:21.800-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:21.811-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:21.810-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:21.811-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:21.831-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.055 seconds
[2024-05-31T15:22:24.510-0500] {processor.py:161} INFO - Started process (PID=13470) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:24.512-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:24.515-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:24.515-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:24.526-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:24.525-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:24.526-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:24.547-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.051 seconds
[2024-05-31T15:22:27.233-0500] {processor.py:161} INFO - Started process (PID=13472) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:27.235-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:27.238-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:27.237-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:27.248-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:27.247-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:27.249-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:27.268-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.053 seconds
[2024-05-31T15:22:29.573-0500] {processor.py:161} INFO - Started process (PID=13474) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:29.574-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:29.577-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:29.577-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:29.583-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:29.582-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 15
    print("hello {name}".for)
                         ^^^
SyntaxError: invalid syntax
[2024-05-31T15:22:29.583-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:29.606-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.050 seconds
[2024-05-31T15:22:31.765-0500] {processor.py:161} INFO - Started process (PID=13477) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:31.766-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:31.769-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:31.769-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:31.777-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:31.776-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:31.777-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:31.797-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.045 seconds
[2024-05-31T15:22:33.359-0500] {processor.py:161} INFO - Started process (PID=13479) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:33.362-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:33.365-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:33.364-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:33.375-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:33.374-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:33.375-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:33.395-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.059 seconds
[2024-05-31T15:22:36.241-0500] {processor.py:161} INFO - Started process (PID=13483) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:36.244-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:36.249-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:36.248-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:36.272-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:36.270-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 29, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:36.273-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:36.312-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.095 seconds
[2024-05-31T15:22:48.614-0500] {processor.py:161} INFO - Started process (PID=13489) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:48.617-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:48.620-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:48.619-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:48.632-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:48.631-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:48.633-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:48.653-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.056 seconds
[2024-05-31T15:22:52.149-0500] {processor.py:161} INFO - Started process (PID=13492) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:52.153-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:52.155-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:52.155-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:52.164-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:52.163-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:52.164-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:52.184-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.046 seconds
[2024-05-31T15:22:56.571-0500] {processor.py:161} INFO - Started process (PID=13495) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:56.572-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:22:56.575-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:56.574-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:56.583-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:22:56.582-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:22:56.583-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:22:56.602-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.049 seconds
[2024-05-31T15:23:02.035-0500] {processor.py:161} INFO - Started process (PID=13499) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:02.036-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:02.038-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:02.037-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:02.046-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:02.045-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:02.046-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:02.065-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.039 seconds
[2024-05-31T15:23:07.731-0500] {processor.py:161} INFO - Started process (PID=13502) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:07.731-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:07.733-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:07.733-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:07.740-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:07.739-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:07.740-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:07.759-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.034 seconds
[2024-05-31T15:23:11.096-0500] {processor.py:161} INFO - Started process (PID=13504) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:11.097-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:11.098-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:11.098-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:11.106-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:11.105-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:11.106-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:11.126-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.037 seconds
[2024-05-31T15:23:16.741-0500] {processor.py:161} INFO - Started process (PID=13507) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:16.742-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:16.743-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:16.743-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:16.750-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:16.749-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:16.750-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:16.769-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.036 seconds
[2024-05-31T15:23:18.775-0500] {processor.py:161} INFO - Started process (PID=13510) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:18.776-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:18.777-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:18.777-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:18.786-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:18.785-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:18.786-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:18.807-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.041 seconds
[2024-05-31T15:23:20.236-0500] {processor.py:161} INFO - Started process (PID=13512) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:20.238-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:20.240-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:20.240-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:20.249-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:20.248-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:20.250-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:20.269-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.051 seconds
[2024-05-31T15:23:24.201-0500] {processor.py:161} INFO - Started process (PID=13515) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:24.201-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:24.203-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:24.202-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:24.210-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:24.209-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:24.210-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:24.229-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.034 seconds
[2024-05-31T15:23:41.595-0500] {processor.py:161} INFO - Started process (PID=13523) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:41.598-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:41.600-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:41.599-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:41.609-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:41.607-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:41.609-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:41.629-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.049 seconds
[2024-05-31T15:23:49.102-0500] {processor.py:161} INFO - Started process (PID=13528) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:49.103-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:49.107-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:49.106-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:49.120-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:49.120-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:49.121-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:49.143-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.053 seconds
[2024-05-31T15:23:51.706-0500] {processor.py:161} INFO - Started process (PID=13531) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:51.708-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:51.712-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:51.711-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:51.723-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:51.722-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:51.723-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:51.744-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.056 seconds
[2024-05-31T15:23:56.944-0500] {processor.py:161} INFO - Started process (PID=13534) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:56.946-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:56.949-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:56.949-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:56.959-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:56.958-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:56.960-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:56.980-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.054 seconds
[2024-05-31T15:23:59.592-0500] {processor.py:161} INFO - Started process (PID=13538) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:59.594-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:23:59.597-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:59.596-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:59.607-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:23:59.606-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 33, in <module>
    python_callable = print_function
NameError: name 'print_function' is not defined
[2024-05-31T15:23:59.607-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:23:59.627-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.050 seconds
[2024-05-31T15:24:02.927-0500] {processor.py:161} INFO - Started process (PID=13541) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:02.928-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:02.930-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:02.930-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:02.934-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:02.933-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 34
    )
    ^
SyntaxError: invalid syntax
[2024-05-31T15:24:02.934-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:02.956-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.036 seconds
[2024-05-31T15:24:05.800-0500] {processor.py:161} INFO - Started process (PID=13543) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:05.801-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:05.802-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:05.802-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:05.813-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:05.900-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:05.900-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:24:05.917-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:05.917-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:24:05.949-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.155 seconds
[2024-05-31T15:24:12.993-0500] {processor.py:161} INFO - Started process (PID=13546) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:12.998-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:13.002-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:13.001-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:13.010-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:13.009-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 35
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:24:13.011-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:13.033-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.057 seconds
[2024-05-31T15:24:19.408-0500] {processor.py:161} INFO - Started process (PID=13551) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:19.409-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:19.411-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:19.410-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:19.414-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:19.413-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 35
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:24:19.414-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:19.434-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.032 seconds
[2024-05-31T15:24:22.002-0500] {processor.py:161} INFO - Started process (PID=13553) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:22.003-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:22.005-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:22.004-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:22.007-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:22.007-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 35
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:24:22.008-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:22.028-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.031 seconds
[2024-05-31T15:24:28.185-0500] {processor.py:161} INFO - Started process (PID=13558) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:28.188-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:28.190-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:28.190-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:28.202-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:28.210-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:28.210-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:24:28.233-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:28.232-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:24:28.273-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.100 seconds
[2024-05-31T15:24:32.179-0500] {processor.py:161} INFO - Started process (PID=13562) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:32.180-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:32.184-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:32.183-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:32.200-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:32.207-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:32.207-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:24:32.227-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:32.227-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:24:32.250-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.089 seconds
[2024-05-31T15:24:34.441-0500] {processor.py:161} INFO - Started process (PID=13565) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:34.442-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:34.445-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:34.445-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:34.449-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:34.448-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 34
    op_kwargs = {'name' : }
                        ^
SyntaxError: expression expected after dictionary key and ':'
[2024-05-31T15:24:34.450-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:34.472-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.039 seconds
[2024-05-31T15:24:40.188-0500] {processor.py:161} INFO - Started process (PID=13569) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:40.189-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:40.191-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:40.191-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:40.203-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:40.298-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:40.298-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:24:40.315-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:40.315-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:24:40.354-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.175 seconds
[2024-05-31T15:24:48.297-0500] {processor.py:161} INFO - Started process (PID=13572) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:48.298-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:48.299-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:48.299-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:48.302-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:48.301-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37
    taskB = 
            ^
SyntaxError: invalid syntax
[2024-05-31T15:24:48.302-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:48.322-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.032 seconds
[2024-05-31T15:24:50.537-0500] {processor.py:161} INFO - Started process (PID=13578) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:50.538-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:50.541-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:50.540-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:50.551-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:50.550-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = py
NameError: name 'py' is not defined
[2024-05-31T15:24:50.551-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:50.571-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.049 seconds
[2024-05-31T15:24:53.840-0500] {processor.py:161} INFO - Started process (PID=13580) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:53.842-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:53.845-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:53.844-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:53.860-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:53.858-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = PythonOperator(
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-05-31T15:24:53.860-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:53.883-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.060 seconds
[2024-05-31T15:24:55.719-0500] {processor.py:161} INFO - Started process (PID=13584) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:55.721-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:55.727-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:55.726-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:55.740-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:55.739-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 38, in <module>
    ta
NameError: name 'ta' is not defined
[2024-05-31T15:24:55.740-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:55.761-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.060 seconds
[2024-05-31T15:24:56.868-0500] {processor.py:161} INFO - Started process (PID=13586) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:56.869-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:56.871-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:56.871-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:56.881-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:56.880-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 38, in <module>
    task_id
NameError: name 'task_id' is not defined
[2024-05-31T15:24:56.881-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:56.901-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.040 seconds
[2024-05-31T15:24:58.598-0500] {processor.py:161} INFO - Started process (PID=13588) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:58.602-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:24:58.607-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:58.606-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:58.612-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:24:58.611-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 39
    )
    ^
SyntaxError: invalid syntax
[2024-05-31T15:24:58.613-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:24:58.635-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.054 seconds
[2024-05-31T15:25:01.957-0500] {processor.py:161} INFO - Started process (PID=13590) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:01.959-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:01.962-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:01.961-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:01.975-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:01.974-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = PythonOperator(
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-05-31T15:25:01.976-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:01.996-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.052 seconds
[2024-05-31T15:25:03.124-0500] {processor.py:161} INFO - Started process (PID=13592) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:03.127-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:03.131-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:03.130-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:03.144-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:03.143-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = PythonOperator(
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-05-31T15:25:03.145-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:03.165-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.060 seconds
[2024-05-31T15:25:04.652-0500] {processor.py:161} INFO - Started process (PID=13593) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:04.653-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:04.654-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:04.654-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:04.664-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:04.662-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = PythonOperator(
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-05-31T15:25:04.664-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:04.683-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.039 seconds
[2024-05-31T15:25:13.110-0500] {processor.py:161} INFO - Started process (PID=13598) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:13.111-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:13.113-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:13.112-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:13.122-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:13.120-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37, in <module>
    taskB = PythonOperator(
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-05-31T15:25:13.122-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:13.141-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.038 seconds
[2024-05-31T15:25:18.486-0500] {processor.py:161} INFO - Started process (PID=13602) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:18.487-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:18.490-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:18.489-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:18.495-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:18.494-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 40
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:25:18.495-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:18.516-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.049 seconds
[2024-05-31T15:25:19.813-0500] {processor.py:161} INFO - Started process (PID=13605) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:19.814-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:19.816-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:19.815-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:19.818-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:19.818-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 40
    )
    ^
SyntaxError: invalid syntax
[2024-05-31T15:25:19.819-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:19.838-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.031 seconds
[2024-05-31T15:25:23.204-0500] {processor.py:161} INFO - Started process (PID=13608) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:23.205-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:23.207-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:23.207-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:23.222-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:23.230-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:23.229-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:23.248-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:23.248-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:23.278-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.083 seconds
[2024-05-31T15:25:24.379-0500] {processor.py:161} INFO - Started process (PID=13610) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:24.381-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:24.385-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:24.384-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:24.402-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:24.410-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:24.410-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:24.429-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:24.429-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:24.451-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.090 seconds
[2024-05-31T15:25:30.649-0500] {processor.py:161} INFO - Started process (PID=13614) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:30.650-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:30.651-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:30.651-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:30.655-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:30.654-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 41
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:25:30.655-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:30.674-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.034 seconds
[2024-05-31T15:25:32.328-0500] {processor.py:161} INFO - Started process (PID=13618) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:32.330-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:32.334-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:32.333-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:32.341-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:32.339-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 41
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:25:32.341-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:32.364-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.054 seconds
[2024-05-31T15:25:34.877-0500] {processor.py:161} INFO - Started process (PID=13620) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:34.878-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:34.883-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:34.882-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:34.898-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:34.905-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:34.905-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:34.924-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:34.924-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:34.965-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.106 seconds
[2024-05-31T15:25:36.147-0500] {processor.py:161} INFO - Started process (PID=13622) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:36.148-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:36.150-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:36.150-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:36.167-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:36.174-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:36.174-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:36.193-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:36.193-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:36.216-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.080 seconds
[2024-05-31T15:25:39.647-0500] {processor.py:161} INFO - Started process (PID=13626) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:39.649-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:39.652-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:39.651-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:39.666-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:39.675-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:39.675-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:39.695-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:39.695-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:39.719-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.097 seconds
[2024-05-31T15:25:41.380-0500] {processor.py:161} INFO - Started process (PID=13628) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:41.381-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:41.383-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:41.382-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:41.386-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:41.385-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 41
    'name' : 
           ^
SyntaxError: expression expected after dictionary key and ':'
[2024-05-31T15:25:41.386-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:41.405-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.034 seconds
[2024-05-31T15:25:45.685-0500] {processor.py:161} INFO - Started process (PID=13630) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:45.686-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:45.688-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:45.687-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:45.699-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:45.706-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:45.706-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:45.725-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:45.725-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:45.760-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.081 seconds
[2024-05-31T15:25:47.588-0500] {processor.py:161} INFO - Started process (PID=13633) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:47.589-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:47.590-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:47.590-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:47.602-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:47.694-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:47.693-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:47.709-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:47.709-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:47.743-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.162 seconds
[2024-05-31T15:25:50.015-0500] {processor.py:161} INFO - Started process (PID=13636) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:50.017-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:50.020-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:50.019-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:50.026-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:50.025-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 42
    'city'
         ^
SyntaxError: ':' expected after dictionary key
[2024-05-31T15:25:50.026-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:50.052-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.049 seconds
[2024-05-31T15:25:54.830-0500] {processor.py:161} INFO - Started process (PID=13639) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:54.831-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:25:54.833-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:54.833-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:54.846-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:25:54.855-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:54.854-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:25:54.874-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:25:54.873-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:25:54.904-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.081 seconds
[2024-05-31T15:26:00.997-0500] {processor.py:161} INFO - Started process (PID=13644) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:00.998-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:26:01.000-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:01.000-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:01.013-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:01.021-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:01.020-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:26:01.041-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:01.041-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:26:01.065-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.081 seconds
[2024-05-31T15:26:31.190-0500] {processor.py:161} INFO - Started process (PID=13662) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:31.195-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:26:31.202-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:31.201-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:31.247-0500] {processor.py:840} INFO - DAG(s) 'python_ops' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:31.266-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:31.266-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:26:31.295-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:31.295-0500] {dag.py:3954} INFO - Setting next_dagrun for python_ops to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:26:31.321-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.157 seconds
[2024-05-31T15:26:53.884-0500] {processor.py:161} INFO - Started process (PID=13684) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:53.886-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:26:53.890-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:53.889-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:53.907-0500] {processor.py:840} INFO - DAG(s) 'python_op' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:26:54.016-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.015-0500] {override.py:1829} INFO - Created Permission View: can delete on DAG:python_op
[2024-05-31T15:26:54.028-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.028-0500] {override.py:1829} INFO - Created Permission View: can edit on DAG:python_op
[2024-05-31T15:26:54.039-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.038-0500] {override.py:1829} INFO - Created Permission View: can read on DAG:python_op
[2024-05-31T15:26:54.039-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.039-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:26:54.048-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.048-0500] {dag.py:3118} INFO - Creating ORM DAG for python_op
[2024-05-31T15:26:54.056-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:26:54.056-0500] {dag.py:3954} INFO - Setting next_dagrun for python_op to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:26:54.079-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.207 seconds
[2024-05-31T15:27:24.276-0500] {processor.py:161} INFO - Started process (PID=13711) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:24.280-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:27:24.285-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:24.284-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:24.309-0500] {processor.py:840} INFO - DAG(s) 'python_op' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:24.331-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:24.330-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:27:24.349-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:24.349-0500] {dag.py:3954} INFO - Setting next_dagrun for python_op to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:27:24.371-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.101 seconds
[2024-05-31T15:27:25.553-0500] {processor.py:161} INFO - Started process (PID=13715) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:25.555-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:27:25.558-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.557-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:25.573-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:25.677-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.676-0500] {override.py:1829} INFO - Created Permission View: can delete on DAG:python_operators_with
[2024-05-31T15:27:25.697-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.696-0500] {override.py:1829} INFO - Created Permission View: can edit on DAG:python_operators_with
[2024-05-31T15:27:25.712-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.711-0500] {override.py:1829} INFO - Created Permission View: can read on DAG:python_operators_with
[2024-05-31T15:27:25.712-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.712-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:27:25.724-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.724-0500] {dag.py:3118} INFO - Creating ORM DAG for python_operators_with
[2024-05-31T15:27:25.733-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:25.732-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:27:25.756-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.222 seconds
[2024-05-31T15:27:28.285-0500] {processor.py:161} INFO - Started process (PID=13717) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:28.287-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:27:28.289-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.288-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:28.301-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:28.407-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.406-0500] {override.py:1829} INFO - Created Permission View: can delete on DAG:python_operators_with_param
[2024-05-31T15:27:28.423-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.422-0500] {override.py:1829} INFO - Created Permission View: can edit on DAG:python_operators_with_param
[2024-05-31T15:27:28.440-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.440-0500] {override.py:1829} INFO - Created Permission View: can read on DAG:python_operators_with_param
[2024-05-31T15:27:28.441-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.440-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:27:28.452-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.452-0500] {dag.py:3118} INFO - Creating ORM DAG for python_operators_with_param
[2024-05-31T15:27:28.460-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:28.460-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:27:28.481-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.204 seconds
[2024-05-31T15:27:29.652-0500] {processor.py:161} INFO - Started process (PID=13722) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:29.653-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:27:29.654-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:29.654-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:29.665-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:27:29.673-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:29.673-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:27:29.693-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:27:29.692-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:27:29.717-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.072 seconds
[2024-05-31T15:28:00.098-0500] {processor.py:161} INFO - Started process (PID=13728) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:00.102-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:00.105-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:00.104-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:00.119-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:00.139-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:00.139-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:28:00.157-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:00.157-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:28:00.180-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.101 seconds
[2024-05-31T15:28:08.869-0500] {processor.py:161} INFO - Started process (PID=13734) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:08.870-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:08.872-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:08.872-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:08.877-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:08.876-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37
    taskB = PythonOperator(
                          ^
SyntaxError: '(' was never closed
[2024-05-31T15:28:08.878-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:08.900-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.043 seconds
[2024-05-31T15:28:10.236-0500] {processor.py:161} INFO - Started process (PID=13736) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:10.237-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:10.240-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:10.239-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:10.244-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:10.243-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37
    taskB = PythonOperator(
                          ^
SyntaxError: '(' was never closed
[2024-05-31T15:28:10.244-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:10.263-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.039 seconds
[2024-05-31T15:28:11.402-0500] {processor.py:161} INFO - Started process (PID=13738) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:11.404-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:11.409-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:11.408-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:11.414-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:11.413-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 37
    taskB = PythonOperator(
                          ^
SyntaxError: '(' was never closed
[2024-05-31T15:28:11.414-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:11.435-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.051 seconds
[2024-05-31T15:28:15.308-0500] {processor.py:161} INFO - Started process (PID=13740) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:15.310-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:15.313-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:15.313-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:15.347-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:15.380-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:15.380-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:28:15.399-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:15.399-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:28:15.436-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.145 seconds
[2024-05-31T15:28:16.534-0500] {processor.py:161} INFO - Started process (PID=13741) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:16.535-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:16.536-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:16.536-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:16.546-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:16.545-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 46, in <module>
    tas
NameError: name 'tas' is not defined
[2024-05-31T15:28:16.547-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:16.568-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.042 seconds
[2024-05-31T15:28:21.277-0500] {processor.py:161} INFO - Started process (PID=13746) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:21.281-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:21.284-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:21.284-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:21.299-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:21.298-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 46, in <module>
    taskA >> ta
NameError: name 'ta' is not defined
[2024-05-31T15:28:21.300-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:21.320-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.057 seconds
[2024-05-31T15:28:23.479-0500] {processor.py:161} INFO - Started process (PID=13750) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:23.479-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:23.481-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:23.480-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:23.492-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:23.581-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:23.580-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:28:23.596-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:23.596-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:28:23.637-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.164 seconds
[2024-05-31T15:28:54.654-0500] {processor.py:161} INFO - Started process (PID=13754) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:54.658-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:28:54.660-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:54.660-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:54.671-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:28:54.692-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:54.692-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:28:54.709-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:28:54.709-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:28:54.732-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.084 seconds
[2024-05-31T15:29:24.967-0500] {processor.py:161} INFO - Started process (PID=13758) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:24.970-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:29:24.971-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:24.971-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:24.984-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:25.004-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:25.004-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:29:25.021-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:25.021-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:29:25.045-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:29:55.213-0500] {processor.py:161} INFO - Started process (PID=13765) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:55.216-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:29:55.217-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:55.217-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:55.228-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:29:55.248-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:55.247-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:29:55.265-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:29:55.265-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:29:55.290-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.082 seconds
[2024-05-31T15:30:25.557-0500] {processor.py:161} INFO - Started process (PID=13772) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:25.561-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:30:25.562-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:25.562-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:25.574-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:25.594-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:25.594-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:30:25.611-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:25.611-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:30:25.638-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:30:56.211-0500] {processor.py:161} INFO - Started process (PID=13782) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:56.213-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:30:56.215-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:56.214-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:56.233-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:30:56.253-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:56.253-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:30:56.270-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:30:56.270-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:30:56.295-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.100 seconds
[2024-05-31T15:31:26.696-0500] {processor.py:161} INFO - Started process (PID=13787) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:26.701-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:31:26.703-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:26.702-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:26.729-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:26.751-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:26.751-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:31:26.769-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:26.769-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:31:26.792-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.101 seconds
[2024-05-31T15:31:57.045-0500] {processor.py:161} INFO - Started process (PID=13794) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:57.045-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:31:57.046-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:57.046-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:57.057-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:31:57.077-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:57.077-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:31:57.095-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:31:57.095-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:31:57.119-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.080 seconds
[2024-05-31T15:32:27.761-0500] {processor.py:161} INFO - Started process (PID=13803) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:27.766-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:32:27.769-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:27.767-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:27.789-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:27.811-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:27.811-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:32:27.830-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:27.829-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-30 00:00:00+00:00, run_after=2024-05-31 00:00:00+00:00
[2024-05-31T15:32:27.861-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.118 seconds
[2024-05-31T15:32:59.009-0500] {processor.py:161} INFO - Started process (PID=13814) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:59.012-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:32:59.014-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:59.013-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:59.033-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:32:59.058-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:59.057-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:32:59.080-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:32:59.080-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:32:59.123-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.127 seconds
[2024-05-31T15:33:30.065-0500] {processor.py:161} INFO - Started process (PID=13820) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:33:30.067-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:33:30.068-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:33:30.068-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:33:30.087-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:33:30.109-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:33:30.109-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:33:30.128-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:33:30.128-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:33:30.157-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.112 seconds
[2024-05-31T15:34:01.156-0500] {processor.py:161} INFO - Started process (PID=13827) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:01.162-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:01.166-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:01.165-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:01.195-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:01.220-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:01.220-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:34:01.240-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:01.240-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:34:01.274-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.144 seconds
[2024-05-31T15:34:31.420-0500] {processor.py:161} INFO - Started process (PID=13840) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:31.424-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:31.426-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:31.425-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:31.453-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:31.475-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:31.475-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:34:31.492-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:31.492-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:34:31.516-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.102 seconds
[2024-05-31T15:34:46.580-0500] {processor.py:161} INFO - Started process (PID=13849) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:46.581-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:46.582-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:46.582-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:46.587-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:46.586-0500] {dagbag.py:355} ERROR - Failed to import: /home/diallo/airflow/dags/execute_with_params.py
Traceback (most recent call last):
  File "/home/diallo/Documents/workspace/Python/AirFlowProjects/AirFlow01/venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/diallo/airflow/dags/execute_with_params.py", line 19
    print("hello {name} nice to see you at {city}".format(name=name, city))
                                                                         ^
SyntaxError: positional argument follows keyword argument
[2024-05-31T15:34:46.587-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:46.607-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.042 seconds
[2024-05-31T15:34:50.583-0500] {processor.py:161} INFO - Started process (PID=13852) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:50.585-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:50.587-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:50.586-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:50.605-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:50.625-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:50.625-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:34:50.643-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:50.642-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:34:50.683-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.112 seconds
[2024-05-31T15:34:51.857-0500] {processor.py:161} INFO - Started process (PID=13854) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:51.859-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:51.860-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:51.859-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:51.876-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:51.898-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:51.898-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:34:51.917-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:51.917-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:34:51.955-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.108 seconds
[2024-05-31T15:34:56.709-0500] {processor.py:161} INFO - Started process (PID=13859) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:56.710-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:34:56.713-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:56.712-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:56.733-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:34:56.760-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:56.760-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:34:56.789-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:34:56.788-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:34:56.820-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.123 seconds
[2024-05-31T15:35:27.901-0500] {processor.py:161} INFO - Started process (PID=13870) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:27.903-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:35:27.904-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:27.904-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:27.915-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:27.935-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:27.935-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:35:27.954-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:27.954-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:35:27.978-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.084 seconds
[2024-05-31T15:35:58.125-0500] {processor.py:161} INFO - Started process (PID=13878) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:58.126-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:35:58.127-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:58.126-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:58.138-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:35:58.158-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:58.157-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:35:58.176-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:35:58.176-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:35:58.205-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.085 seconds
[2024-05-31T15:36:28.633-0500] {processor.py:161} INFO - Started process (PID=13883) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:28.636-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:36:28.637-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:28.636-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:28.648-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:28.668-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:28.668-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:36:28.686-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:28.686-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:36:28.709-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.082 seconds
[2024-05-31T15:36:59.299-0500] {processor.py:161} INFO - Started process (PID=13890) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:59.302-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:36:59.303-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:59.303-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:59.315-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:36:59.335-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:59.335-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:36:59.353-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:36:59.353-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:36:59.384-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.091 seconds
[2024-05-31T15:37:30.350-0500] {processor.py:161} INFO - Started process (PID=13895) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:37:30.351-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:37:30.352-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:37:30.351-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:37:30.368-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:37:30.394-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:37:30.393-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:37:30.416-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:37:30.416-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:37:30.445-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.103 seconds
[2024-05-31T15:38:01.574-0500] {processor.py:161} INFO - Started process (PID=13905) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:01.578-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:38:01.580-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:01.579-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:01.597-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:01.620-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:01.620-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:38:01.639-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:01.639-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:38:01.665-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.113 seconds
[2024-05-31T15:38:32.695-0500] {processor.py:161} INFO - Started process (PID=13910) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:32.700-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:38:32.703-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:32.702-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:32.737-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:38:32.765-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:32.764-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:38:32.786-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:38:32.786-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:38:32.816-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.142 seconds
[2024-05-31T15:39:03.490-0500] {processor.py:161} INFO - Started process (PID=13915) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:03.495-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:39:03.497-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:03.496-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:03.527-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:03.569-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:03.569-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:39:03.593-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:03.592-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:39:03.618-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.151 seconds
[2024-05-31T15:39:33.942-0500] {processor.py:161} INFO - Started process (PID=13920) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:33.947-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:39:33.952-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:33.950-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:33.988-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:39:34.017-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:34.017-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:39:34.040-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:39:34.040-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:39:34.066-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.150 seconds
[2024-05-31T15:40:04.499-0500] {processor.py:161} INFO - Started process (PID=13930) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:04.501-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:40:04.503-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:04.502-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:04.519-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:04.539-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:04.539-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:40:04.558-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:04.558-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:40:04.582-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.101 seconds
[2024-05-31T15:40:34.762-0500] {processor.py:161} INFO - Started process (PID=13935) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:34.766-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:40:34.767-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:34.767-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:34.782-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:40:34.804-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:34.804-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:40:34.822-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:40:34.822-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:40:34.852-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.096 seconds
[2024-05-31T15:41:05.064-0500] {processor.py:161} INFO - Started process (PID=13942) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:05.065-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:41:05.066-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:05.065-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:05.077-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:05.097-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:05.096-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:41:05.114-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:05.114-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:41:05.145-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:41:35.418-0500] {processor.py:161} INFO - Started process (PID=13946) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:35.422-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:41:35.425-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:35.424-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:35.452-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:41:35.474-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:35.474-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:41:35.492-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:41:35.491-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:41:35.515-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.102 seconds
[2024-05-31T15:42:05.807-0500] {processor.py:161} INFO - Started process (PID=13953) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:05.807-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:42:05.808-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:05.808-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:05.822-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:05.842-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:05.842-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:42:05.859-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:05.859-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:42:05.883-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.083 seconds
[2024-05-31T15:42:36.110-0500] {processor.py:161} INFO - Started process (PID=13959) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:36.113-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:42:36.114-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:36.114-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:36.126-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:42:36.147-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:36.146-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:42:36.164-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:42:36.164-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:42:36.190-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:43:06.443-0500] {processor.py:161} INFO - Started process (PID=13966) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:06.447-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:43:06.449-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:06.448-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:06.476-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:06.498-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:06.498-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:43:06.517-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:06.517-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:43:06.539-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.102 seconds
[2024-05-31T15:43:36.741-0500] {processor.py:161} INFO - Started process (PID=13970) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:36.745-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:43:36.746-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:36.745-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:36.759-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:43:36.780-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:36.780-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:43:36.798-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:43:36.798-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:43:36.821-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:44:07.046-0500] {processor.py:161} INFO - Started process (PID=13977) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:07.049-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:44:07.050-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:07.050-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:07.062-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:07.082-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:07.082-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:44:07.100-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:07.100-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:44:07.125-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.085 seconds
[2024-05-31T15:44:37.451-0500] {processor.py:161} INFO - Started process (PID=14077) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:37.452-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:44:37.453-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:37.453-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:37.473-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:44:37.502-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:37.502-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:44:37.526-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:44:37.526-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:44:37.567-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.126 seconds
[2024-05-31T15:45:07.838-0500] {processor.py:161} INFO - Started process (PID=14093) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:07.843-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:45:07.846-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:07.845-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:07.866-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:07.887-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:07.887-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:45:07.905-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:07.905-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:45:07.936-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.114 seconds
[2024-05-31T15:45:38.205-0500] {processor.py:161} INFO - Started process (PID=14099) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:38.206-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:45:38.208-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:38.207-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:38.223-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:45:38.243-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:38.243-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:45:38.261-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:45:38.261-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:45:38.285-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.099 seconds
[2024-05-31T15:46:08.636-0500] {processor.py:161} INFO - Started process (PID=14104) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:08.641-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:46:08.643-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:08.642-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:08.661-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:08.681-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:08.681-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:46:08.700-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:08.700-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:46:08.737-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.119 seconds
[2024-05-31T15:46:39.002-0500] {processor.py:161} INFO - Started process (PID=14108) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:39.004-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:46:39.005-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:39.005-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:39.020-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:46:39.040-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:39.040-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:46:39.057-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:46:39.057-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:46:39.083-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.097 seconds
[2024-05-31T15:47:09.369-0500] {processor.py:161} INFO - Started process (PID=14115) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:09.372-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:47:09.374-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:09.374-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:09.386-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:09.410-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:09.410-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:47:09.431-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:09.431-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:47:09.455-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.096 seconds
[2024-05-31T15:47:39.767-0500] {processor.py:161} INFO - Started process (PID=14123) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:39.768-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:47:39.769-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:39.768-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:39.780-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:47:39.800-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:39.799-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:47:39.818-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:47:39.818-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:47:39.848-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:48:10.427-0500] {processor.py:161} INFO - Started process (PID=14219) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:10.433-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:48:10.435-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:10.434-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:10.460-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:10.484-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:10.484-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:48:10.511-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:10.511-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:48:10.542-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.141 seconds
[2024-05-31T15:48:40.623-0500] {processor.py:161} INFO - Started process (PID=14223) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:40.627-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:48:40.628-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:40.628-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:40.639-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:48:40.659-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:40.659-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:48:40.677-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:48:40.677-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:48:40.702-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.085 seconds
[2024-05-31T15:49:11.252-0500] {processor.py:161} INFO - Started process (PID=14251) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:11.253-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:49:11.255-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:11.254-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:11.268-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:11.290-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:11.289-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:49:11.310-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:11.309-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:49:11.339-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.099 seconds
[2024-05-31T15:49:41.780-0500] {processor.py:161} INFO - Started process (PID=14303) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:41.785-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:49:41.788-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:41.786-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:41.812-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:49:41.833-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:41.833-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:49:41.852-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:49:41.851-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:49:41.889-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.128 seconds
[2024-05-31T15:50:12.110-0500] {processor.py:161} INFO - Started process (PID=14320) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:12.113-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:50:12.114-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:12.114-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:12.125-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:12.145-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:12.145-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:50:12.163-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:12.163-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:50:12.188-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.083 seconds
[2024-05-31T15:50:42.531-0500] {processor.py:161} INFO - Started process (PID=14325) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:42.533-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:50:42.535-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:42.535-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:42.559-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:50:42.588-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:42.587-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:50:42.611-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:50:42.611-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:50:42.639-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.134 seconds
[2024-05-31T15:51:12.969-0500] {processor.py:161} INFO - Started process (PID=14333) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:12.970-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:51:12.972-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:12.972-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:12.991-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:13.016-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:13.016-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:51:13.040-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:13.040-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:51:13.083-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.123 seconds
[2024-05-31T15:51:43.389-0500] {processor.py:161} INFO - Started process (PID=14359) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:43.392-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:51:43.393-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:43.392-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:43.405-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:51:43.425-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:43.425-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:51:43.444-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:51:43.444-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:51:43.469-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.086 seconds
[2024-05-31T15:52:13.697-0500] {processor.py:161} INFO - Started process (PID=14367) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:13.701-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:52:13.702-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:13.701-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:13.712-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:13.733-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:13.733-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:52:13.751-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:13.751-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:52:13.772-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.081 seconds
[2024-05-31T15:52:44.150-0500] {processor.py:161} INFO - Started process (PID=14393) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:44.154-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:52:44.157-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:44.156-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:44.194-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:52:44.223-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:44.222-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:52:44.241-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:52:44.241-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:52:44.264-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.133 seconds
[2024-05-31T15:53:14.435-0500] {processor.py:161} INFO - Started process (PID=14402) to work on /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:53:14.438-0500] {processor.py:830} INFO - Processing file /home/diallo/airflow/dags/execute_with_params.py for tasks to queue
[2024-05-31T15:53:14.439-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:53:14.439-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:53:14.451-0500] {processor.py:840} INFO - DAG(s) 'python_operators_with_param' retrieved from /home/diallo/airflow/dags/execute_with_params.py
[2024-05-31T15:53:14.471-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:53:14.471-0500] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-31T15:53:14.489-0500] {logging_mixin.py:188} INFO - [2024-05-31T15:53:14.489-0500] {dag.py:3954} INFO - Setting next_dagrun for python_operators_with_param to 2024-05-31 00:00:00+00:00, run_after=2024-06-01 00:00:00+00:00
[2024-05-31T15:53:14.514-0500] {processor.py:183} INFO - Processing /home/diallo/airflow/dags/execute_with_params.py took 0.085 seconds
